plugins/katello/3.4/release_notes/release_notes.md

```diff
@@ -4,12 +4,28 @@ title: Release Notes
 version: 3.4
 ---
 
-# Katello Nightly () Release Notes
+# Katello 3.4 (Oud bruin) Release Notes
 
-For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/master/CHANGELOG.md).
+For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/KATELLO-3.4/CHANGELOG.md).
 
 ## Features
 
+#### File Repository Additions
+
+#### Repository Force Sync
+
+#### Content View Force Republish
+
+#### Subscription Updates
+
+#### Docker Repository Discovery
+
+#### UI Overhaul
```

Comment:
@waldenraines Could you do a write up of all the various UI overhauls that are being incorporated with 3.4? Thinking things like switching to paginated tables, nutupane removal, etc. If you want to include any screenshots throw in comments and I'll copy them to the PR.